README.rst

```diff
@@ -92,26 +93,38 @@ We assume you may have enough storage space on your PC.
 How to add your code to code base
```

Comment:
A reader may not be the owner of the code.

- How to add a code to the code base
- How to add a project to the code base
- How to add a git project to the code base

The last one don't not require the next sentence.